client/src/common/components/Form.jsx

```diff
@@ -1,6 +1,6 @@
 import React, { PureComponent } from 'react';
 import PropTypes from 'prop-types';
-import { injectIntl } from 'react-intl';
+import { injectIntl, FormattedMessage } from 'react-intl';
```

Comment:
Consider sorting this. In order to minimize nit-picking, consider using https://eslint.org/docs/rules/sort-imports

Reply:
I will add this rule on a separate branch(task), because there will too many diffs to check on this PR